src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java

```diff
@@ -1222,19 +1223,20 @@ public Path zipRepositoryWithParticipation(Repository repo, String repositoryDir
         else {
             zipRepoName += "-" + studentTeamOrDefault + ".zip";
         }
-        return zipRepository(repo, zipRepoName, repositoryDir);
+        return zipRepository(repo, zipRepoName, repositoryDir, null);
     }
 
     /**
-     * Zips the contents of a git repository.
+     * Zips the contents of a git repository with or without ".git" directory.
```

Comment:
What do you mean by this comment? If this just refers to `contentFilter`, I think you could write that in general.

Reply:
Yes, it is for the `contentFilter`, since the generic explanation for the parameter is given in the `@param` I wanted to include a intended use case here but yes it makes sense to make the wording more clear.